packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js

```diff
@@ -5,16 +5,27 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
   $rootScope,
   Alerter,
   IpGameFirewall,
+  atInternet,
 ) {
   const self = this;
   const alert = 'ip_game_firewall_alert';
 
   self.datas = $scope.currentActionData;
   self.loading = false;
 
+  atInternet.trackClick({
+    name: $scope.data?.tracking,
+    type: 'action',
+  });
+
   $scope.removeGameFirewallRule = function removeGameFirewallRule() {
     self.loading = true;
 
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}-confirm`,
```

Comment:
in that way, you're sending `null-confirm` as a potential value 
```suggestion
      name: $scope.data?.tracking} ? `${$scope.data?.tracking}-confirm` : null,
```